cmd/antrea-agent/agent.go

```diff
@@ -64,17 +65,18 @@ const informerDefaultResync = 12 * time.Hour
 // run starts Antrea agent with the given options and waits for termination signal.
 func run(o *Options) error {
 	klog.Infof("Starting Antrea agent (version %s)", version.GetFullVersion())
+
+	// set up signal capture: the first SIGTERM / SIGINT signal is handled gracefully and will
```

Comment:
I know not your code, but maybe capitalize "set up" to "Set up", for multiple sentences comments.

Reply:
Updated.